docs/user-guide/inference-backend-management.md

```diff
@@ -17,6 +17,10 @@ For guidelines for configuring custom backends and examples of custom backends t
 | Version Configs            | Configure available versions of this backend                                                                                                                                                                                                                                                                                                                                                             | Yes      |
 | Default Version            | Preselected during deployment. If you don’t choose a version, its image is used                                                                                                                                                                                                                                                                                                                          | No       |
 
+!!! note
+
+    Execution command strings are parsed into arguments using POSIX-style rules. Quote values with spaces and avoid shell operators. Placeholders (e.g., `{{model_path}}`, `{{port}}`) are replaced before parsing.
```

Comment:
How about putting it in parameter description. Does the same apply to `Default Execution Command`?